experimental/CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ All notable changes to experimental packages in this project will be documented
 
 ### :bug: (Bug Fix)
 
+* fix(otlp-exporter-base): don't create blob before sending xhr [#5193](https://github.com/open-telemetry/opentelemetry-js/pull/5193)
+  * improves compatibility with some unsupported runtimes
```

Comment:
note: I'm unsure if this should be categorized as a bug or improvment, since we do not claim to support React Native.

Reply:
ya it's a bit odd, there shouldn't be any reason not to use `Blob` for node or web, it's just that React Native has some issues with how it's implemented on there

maybe it's an improvement in the sense that this simplifies the code slightly and removes an unnecessary step in processing the data?

Reply:
Yep - I moved it to enhancement. Thanks 🙌